### PR TITLE
Improve output message with config file creation

### LINKED ISSUE #586: [enhancement] Improve the stdout of the `config init --dest-dir` command
## ~Bug Report~ Enhancement request

The config file creation works as expected, it is not a bug report.

### Current behavior

I spotted a minor inconsistency in the stdout. Preferably, I want to always see the absolute path of the newly created config file:

If I run the `init` command with `.` destination directory:
```
./arduino-cli config init --dest-dir .        
Config file written to: arduino-cli.yaml
```
If I run the `init` command with `~/Desktop` dest dir:
```
./arduino-cli config init --dest-dir ~/Desktop
Config file written to: /Users/akos.kitta/Desktop/arduino-cli.yaml
```

### Expected behavior

```
pwd
/path/to/your/cwd
./arduino-cli config init --dest-dir .        
Config file written to: /path/to/your/cwd/arduino-cli.yaml
```

### Environment

- CLI version (output of `arduino-cli version`): I built it from the source: bc8e0731
- OS and platform: macOS

### PR DESCRIPTION
**Pull request by vinay-lanka**
Improved output message with config file creation 

Fix #586

### CODE REVIEW

**Comment by vinay-lanka** [comment]
Could anyone take a look at this change? 
I'd like to know if I could improve it in any way

File: cli/config/init.go [hunk 1]
```diff
@@ -62,7 +62,14 @@ func runInitCommand(cmd *cobra.Command, args []string) {
 		os.Exit(errorcodes.ErrGeneric)
 	}
 
-	msg := "Config file written to: " + configFile
+	var absPath, err = filepath.Abs(destDir)
+	if err != nil {
+		feedback.Errorf("Cannot find absolute path: %v", err)
+		os.Exit(errorcodes.ErrGeneric)
+	}
+	var configFileAbsPath = filepath.Join(absPath, defaultFileName)
```
Comment (rsora) [inline cli/config/init.go:70]:
```suggestion
	absPath, err := filepath.Abs(destDir)
	if err != nil {
		feedback.Errorf("Cannot find absolute path: %v", err)
		os.Exit(errorcodes.ErrGeneric)
	}
	configFileAbsPath := filepath.Join(absPath, defaultFileName)
```

Comment (rsora) [reply cli/config/init.go:70]: After applying the suggestion, 
- let's move your code up in the function (to line 52, after the `if destDir == "" {` closing)
- replace `destDir` with your `absPath` in the function
- replace `configFile` with your `configFileAbsPath` in the function

This way it should be more clean :+1:

**Comment by vinay-lanka** [comment]
Oh yeah, thanks for pointing it out. I rebased on the latest master.
Glad I was able to contribute! :)

**Comment by rsora** [comment]
Thanks for contributing!

**Pull request merged**